src/cloud_tasks/cli.py

```
@@ -626,252 +470,885 @@ async def list_running_instances_cmd(args: argparse.Namespace, config: Config) -
+        self.task_db = task_db
+        self.output_file_path = output_file_path
+        self.print_events = print_events
+        self.print_summary = print_summary
+        self.output_file = None
+        self.something_changed = True
+
+    async def start(self) -> None:
+        """Start monitoring events."""
+        if self.output_file_path:
+            try:
+                self.output_file = open(self.output_file_path, "a")
```

Ruff (0.14.14)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)